kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -916,4 +916,13 @@ object Stream:
                         case Present(value -> nextAcc) => Loop.continue(curChunk.append(value), nextAcc)
                         case Absent                    => Emit.valueWith(curChunk)(Loop.done(()))
 
+    /** Takes a Stream[V, S] in the context of S2 (i.e. Stream[V, S] < S2) and returns a Stream that fuses together both effect contexts S
+      * and S2 into a single Stream[V, S & S2].
+      *
+      * @param stream
+      *   The stream to unwrap
+      * @return
+      *   A new stream that fuses together both effect contexts S and S2 into a single Stream[V, S & S2]
+      */
+    inline def unwrap[V, S, S2](stream: Stream[V, S] < S2): Stream[V, S & S2] = Stream(stream.map(_.emit))
```

Comment:
how about an extension method? I don't see the need to put it in the combinators module

Reply:
It occurs to me any extension method should probably be `unwrapStream` in case there are other types we want to unwrap. (E.g., Pipe/Sink)